dart/lib/src/event_processor/exception/io_exception_event_processor.dart

```diff
@@ -0,0 +1,94 @@
+import 'dart:io';
+
+import '../../protocol.dart';
+import 'exception_event_processor.dart';
+
+ExceptionEventProcessor exceptionEventProcessor() =>
+    IoExceptionEventProcessor();
+
+class IoExceptionEventProcessor implements ExceptionEventProcessor {
+  @override
+  SentryEvent apply(SentryEvent event, {dynamic hint}) {
```

Comment:
Wondering if there's more exception types that we can enrich the event with.
Good idea.

Reply:
I'm not sure though how much it's worth to try to improve reports for most of those exceptions, though.

Reply:
Yeah, I guess we can leverage this event processor over time depending on its needs.